python/cog/server/log_capture.py

```diff
@@ -75,7 +75,7 @@ def capture_log(logs_dest):
             os.close(old_fds[out_name])
 
 
-class LogProcess(multiprocessing.Process):
+class LogProcess(multiprocessing.get_context("fork").Process):
```

Comment:
Could you add a line of documentation here explaining what `multiprocessing.get_context("fork").Process` does? It's a fairly unusual pattern and we might forget the rationale.

Reply:
Comment added!